en/docs/security-announcements/cve-justifications/2025/CVE-2025-48913.md

```diff
@@ -20,7 +20,9 @@ A vulnerability was reported in Apache CXF affecting the Java Message Service (J
 
 ### WSO2 JUSTIFICATION
 
-WSO2 API Manager (APIM) does not make use of Apache CXF’s JMS transport classes. As a result, the vulnerable code path in Apache CXF is **never exercised** within the APIM runtime. Therefore, none of the APIM versions listed above are affected by this vulnerability.
+WSO2 API Manager (APIM) does **not** make use of the Apache CXF component `org.apache.cxf:cxf-rt-transports-jms`, which is the only module affected by this vulnerability [^2]. Therefore, the **vulnerable code path is not present** or exercised in the APIM runtime.
```

Comment:
@SavinduDimal I feels, if we add [1][2], this clarification will further justify. [1] says vulnerable to this CVE where [2] says it's not. WDYT? 


[1]. https://security.snyk.io/package/maven/org.apache.cxf%3Acxf-rt-transports-jms/3.5.11
[2]. https://security.snyk.io/package/maven/org.apache.cxf%3Acxf-core/3.5.11